addons/sale/data/mail_template_data.xml

```diff
@@ -10,7 +10,9 @@
             <field name="body_html" type="html">
 <div style="margin: 0px; padding: 0px;">
     <p style="margin: 0px; padding: 0px; font-size: 13px;">
-        <t t-set="doc_name" t-value="'quotation' if object.state in ('draft', 'sent') else 'order'"/>
+        <t t-set="quotation">quotation</t>
+        <t t-set="order">order</t>
+        <t t-set="doc_name" t-value="quotation if object.state in ('draft', 'sent') else order"/>
```

Comment:
```suggestion
        <t t-if="object.state in ('draft', 'sent')" t-set="doc_name">quotation</t>
        <t t-else="" t-set="doc_name">order</t>
```
other solution ?
Imho, it can be cleaner if we can avoid 'useless' intermediary variables :thinking: